specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2019-08-01/notebook.json

```diff
@@ -0,0 +1,446 @@
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceList": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/notebookWorkspaces/{notebookWorkspaceName}": {
+      "get": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "description": "Gets the notebook workspace for a Cosmos DB account.",
+        "operationId": "NotebookWorkspaces_Get",
+        "parameters": [
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified notebook workspace resource.",
+            "schema": {
+              "$ref": "#/definitions/NotebookWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-08-01/cosmos-db.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBNotebookWorkspaceGet": {
+            "$ref": "./examples/CosmosDBNotebookWorkspaceGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "NotebookWorkspacesResource"
+        ],
+        "description": "Creates the notebook workspace for a Cosmos DB account.",
+        "operationId": "NotebookWorkspaces_Create",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NotebookWorkspaceNameParameter"
+          },
+          {
+            "name": "createParameters",
```

Comment:
The commonly used word is `parameters`. I don't think `createParameters` is a good naming. Is it able to change it to a meaningful word like `notebookParameter`?